packages/waku/src/lib/middleware/dev-server-impl.ts

```diff
@@ -128,7 +128,7 @@ const createMainViteServer = (
               include: ['react-server-dom-webpack/client.edge'],
             },
           },
-          appType: 'custom',
+          appType: 'mpa',
```

Comment:
"mpa" is a server-rendered multi-page app. and "custom" is a spa with a server for rsc?

Reply:
This actually doesn't matter, but feels more consistent. (I recently changed from 'custom' to 'mpa', hoping to fix something.)